### DISCUSSION TITLE
Welcome to SailwindModRepository Discussions!

### DISCUSSION THREAD

**Discussion by MaxWasUnavailable** [MaxWasUnavailable/SailwindModRepository#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xrami19** [comment]
nice mods, any updates planned, not working in current version.

**Reply by MaxWasUnavailable** [reply]
This is no longer in use, sorry! You can find mods on the Discord server.
